ui/components/app/app-header/app-header.component.js

```diff
@@ -96,31 +103,47 @@ export default class AppHeader extends PureComponent {
     const {
       history,
       isUnlocked,
+      isAccountMenuOpen,
+      networkDropdownOpen,
       hideNetworkIndicator,
       disableNetworkIndicator,
       disabled,
       onClick,
     } = this.props;
+    const pathname = history?.location?.pathname;
+    const logoShow = pathname === UNLOCK_ROUTE;
 
     return (
       <div
         className={classnames('app-header', {
           'app-header--back-drop': isUnlocked,
+          'app-header--filter': isAccountMenuOpen && getEnvironmentType() != ENVIRONMENT_TYPE_FULLSCREEN,
+          'app-header--filter': networkDropdownOpen && getEnvironmentType() != ENVIRONMENT_TYPE_FULLSCREEN,
         })}
       >
         <div className="app-header__contents">
-          <MetaFoxLogo
+          {/* <MetaFoxLogo
+            unsetIconHeight
+            onClick={async () => {
+              if (onClick) {
+                await onClick();
+              }
+              history.push(DEFAULT_ROUTE);
+            }}
+          /> */}
+          {!logoShow ? <AlphaCarbonLogoGradient
```

Comment:
```suggestion
          {isUnlocked &&  <AlphaCarbonLogoGradient
```
- The syntax `isShow && <ItemToShow />` can do the trick for you, instead of using `isShow ? <ItemToShow /> : null`
- Also there is the variable `isUnlocked`, so you don't need `pathname` and `logoShow`

Reply:
fixed in cf4268c5f748a08014cf4f6dae9d7f610a151715